schema/pgsql/upgrades/1.2.0.sql

```diff
@@ -14,3 +14,11 @@ CREATE INDEX idx_servicegroup_name_ci ON servicegroup(name_ci);
 COMMENT ON INDEX idx_servicegroup_display_name IS 'Servicegroup list filtered/ordered by display_name';
 COMMENT ON INDEX idx_servicegroup_name_ci IS 'Servicegroup list filtered using quick search';
 COMMENT ON INDEX idx_servicegroup_name IS 'Host/service/service group list filtered by service group name; Servicegroup detail filter';
+
+ALTER TYPE history_type RENAME TO history_type_old;
+CREATE TYPE history_type AS ENUM ( 'state_change', 'ack_clear', 'comment_remove', 'downtime_end', 'flapping_end', 'ack_set', 'comment_add', 'downtime_start', 'flapping_start', 'notification' );
+ALTER TABLE history
+  ALTER COLUMN event_type DROP DEFAULT,
+  ALTER COLUMN event_type TYPE history_type USING event_type::text::history_type,
+  ALTER COLUMN event_type SET DEFAULT 'state_change'::history_type;
```

Comment:
`UPDATE history SET event_type_new = event_type::text::history_type;` is already running for 15 minutes on the same database and not finished yet, so I don't think this will help.

Reply:
Finished after 70 minutes, so yeah, not the way to go.